htdocs/langs/en_US/banks.lang

```diff
@@ -148,7 +148,7 @@ BackToAccount=Back to account
 ShowAllAccounts=Show for all accounts
 FutureTransaction=Transaction in futur. No way to conciliate.
 SelectChequeTransactionAndGenerate=Select/filter checks to include into the check deposit receipt and click on "Create".
-InputReceiptNumber=Choose the bank statement related with the conciliation. Use a sortable numeric value (such as, YYYYMM)
+InputReceiptNumber=Choose the bank statement related with the conciliation. Use a sortable numeric value: YYYYMM or YYYYMMDD
```

Comment:
1- I think the term "alphanumeric order" is not clear enough for the end user. A lot of dolibarr users just don't know what this means.

2- I can't see what other formats would be sorted correctly actually YYMM doesn't work for instance.